dlrover/python/master/shard_manager/batch_dataset_manager.py

```diff
@@ -187,3 +186,169 @@ def restore_checkpoint(self, checkpoint: DatasetShardCheckpoint):
                     shard,
                 )
             )
+
+
+
+class StreamingDatasetManager(DatasetManger):
```

Comment:
The class is duplicated with streaming_dataset_manager.py

Reply:
The duplicated class is removed.